packages/test/test-end-to-end-tests/src/test/legacyChunking.spec.ts

```diff
@@ -0,0 +1,107 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { SharedMap } from "@fluidframework/map";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import {
+    ITestFluidObject,
+    ChannelFactoryRegistry,
+    ITestObjectProvider,
+    ITestContainerConfig,
+    DataObjectFactoryType,
+    TestFluidObjectFactory,
+} from "@fluidframework/test-utils";
+import { installVersionsDescribe, getContainerRuntimeApi } from "@fluidframework/test-version-utils";
+import { IContainer } from "@fluidframework/container-definitions";
+import { FlushMode, IContainerRuntimeBase } from "@fluidframework/runtime-definitions";
+import { IRequest } from "@fluidframework/core-interfaces";
+
+const versionWithChunking = "0.56.0";
+
+installVersionsDescribe(
+    {
+        requestAbsoluteVersions: [versionWithChunking],
+    }
+)(
+    "Legacy chunking",
+    (getTestObjectProvider) => {
+        let provider: ITestObjectProvider;
+        let oldMap: SharedMap;
+        let newMap: SharedMap;
+        beforeEach(() => {
+            provider = getTestObjectProvider();
+        });
+        afterEach(async () => provider.reset());
+
+        const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+            runtime.IFluidHandleContext.resolveHandle(request);
+        const mapId = "map";
+        const registry: ChannelFactoryRegistry = [
```

Comment:
Note that this will create a SharedMap of current version. Same with the TestFluidObjectFactory.
To get older versions, you will have to use equivalent of getContainerRuntimeApi for these as well.

Reply:
Sure. But that's ok, right? I mean the point is to just use a different runtime, as that's the one who's chunking the op. When it reaches the DDS, it's just an op anyway. The chuncked ops are accumulated/processed in the runtime only.

Reply:
yes, agreed.